presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -288,6 +288,7 @@ public final class SystemSessionProperties
     public static final String PULL_EXPRESSION_FROM_LAMBDA_ENABLED = "pull_expression_from_lambda_enabled";
     public static final String REWRITE_CONSTANT_ARRAY_CONTAINS_TO_IN_EXPRESSION = "rewrite_constant_array_contains_to_in_expression";
     public static final String INFER_INEQUALITY_PREDICATES = "infer_inequality_predicates";
+    public static final String ENABLE_HBO_FOR_SCALED_WRITER = "enable_hbo_for_scaled_writer";
```

Comment:
We try not to use abbreviations in our codebase. We can use "enable_history_based_scaled_writer"